cmd/system-probe/main.go

```diff
@@ -93,6 +93,19 @@ func main() {
 		os.Exit(1)
 	}
 
+	platform, err := util.GetPlatform()
+	if err != nil {
+		log.Debugf("error retrieving platform: %s", err)
+	} else {
+		log.Infof("running on platform: %s", platform)
+	}
+
+	mounted := util.IsDebugfsMounted()
+	if !mounted {
+		log.Info("debugfs is not mounted, run \"sudo mount -t debugfs none /sys/kernel/debug\" to mount debugfs")
```

Comment:
I think it would be useful to add a little more context: "debugfs is not mounted and is needed for eBPF-based checks, ..."

Reply:
Hmm - should can this be done where ever the OS unsupported check is done? I think the order we want is OS check and then debugfs check.